docs/classes/request.md

```diff
@@ -2,86 +2,128 @@
 title: Request
 permalink: /classes/request
 ---
+# Request
+
+Class **`Request`** extends [`Message`](message.md)
+
+The `Request` class represents an HTTP request. It's used in [request handlers](requesthandler.md) and [`middleware`](middleware.md).
```

Comment:
`requesthandler.md` should be `request-handler.md`.

Reply:
Renamed